semcore/mini-chart/src/component/score/Line.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import createComponent, { Component, Root, sstyled, ComponentType } from '@semcore/core';
+import { Box, Flex, BoxProps } from '@semcore/flex-box';
+import resolveColorEnhance from '@semcore/utils/lib/enhances/resolveColorEnhance';
+
+import style from './line.shadow.css';
+
+export type CommonScoreProps = {
```

Comment:
Probably you need to add required `aria-label` prop to require all developers to always care about charts a11y on themselves

Reply:
No, we don't. We told about that with @j-mnizhek and decided that we don't need to add required label.